lib/diagnostic.rb

```diff
@@ -91,16 +108,20 @@ def eat_meat(food)
 end
 
 ##
-# your code here
-##
+class Lion < Cat
+  include Carnivorous
+  def roar
+    puts "ROAR"
+  end
+end##
 
 # Question 6
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 # Write your answer as a comment in the section below.
 
 ##
-# your answer, in comments, here
+# It saves time and make code mobile.
```

Comment:
# In direct inheritance, you're limited to inheriting from a single source.
# In composition, you can draw from as many modules as you like.